components/Dialogs/src/MDCAlertController.h

```diff
@@ -161,6 +161,14 @@
  */
 - (void)addAction:(nonnull MDCAlertAction *)action;
 
+/*
+ Determines if an @c MDCAlertController should use the @c MDCRippleView behavior
+ or not. By setting this property to @c YES, @c MDCRippleView is used to provide the user visual
+ touch feedback, instead of the legacy @c MDCInkView.
+ @note Defaults to @c NO.
+ */
```

Comment:
Similarly here as in https://github.com/material-components/material-components-ios/pull/7596/files#r293630410